pkg/didcomm/dispatcher/api.go

```diff
@@ -30,6 +30,9 @@ type MessageService interface {
 type MessageHandler interface {
 	// Services returns list of available message services in this message handler
 	Services() []MessageService
+	// SetOutbound sets outbound dispatcher to this provider
+	// TODO should use dedicated messenger interface instead of Outbound dispatcher [Issue #1058]
+	SetOutbound(Outbound) error
```

Comment:
We don't use setters in this project. This can be injected when instantiating.

Reply:
this `SetOutbound()` is to set Outbound from framework to injected handler.

Reply:
Or we can call it `InitOutbound(...)` to avoid confusion

Reply:
Is there an expectation to change the `outbound` after instantiation?

Reply:
external code cant use context. This way, I was injecting framework's `outbound` to registrar. I am going to change this.

Reply:
removed `SetOutbound`